### PR TITLE
Server logging 931

### PR DESCRIPTION
**Pull request by swith004**
This PR adds logging to the model connection configuration in order to debug possible issues in the log.

### CODE REVIEW

File: caikit_tgis_backend/tgis_backend.py [hunk 1]
```diff
@@ -101,6 +101,10 @@ def __init__(self, config: Optional[dict] = None):
         # Parse connection objects for all model-specific connections
         for model_id, model_conn_cfg in self._remote_models_cfg.items():
             model_conn = TGISConnection.from_config(model_id, model_conn_cfg)
+
+            log.debug(
+                "model connection config is set to the following: %s", model_conn_cfg
```
Comment (evaline-ju) [inline caikit_tgis_backend/tgis_backend.py]: Since this corresponds to a model would it be helpful to also log the `model_id`?

Comment (swith004) [reply caikit_tgis_backend/tgis_backend.py]: thanks, @evaline-ju  - made the change

**Review by evaline-ju — COMMENTED** [review]
I see the DCO check is stuck but could you [rebase your commits with the signoff](https://cert-manager.io/docs/contributing/sign-off/)? thanks!

**Comment by swith004** [comment]
> I see the DCO check is stuck but could you [rebase your commits with the signoff](https://cert-manager.io/docs/contributing/sign-off/)? thanks!

missed this since update is sent to another email address! Thanks @evaline-ju  - forgot the signoff needed in the caikit repos.

File: caikit_tgis_backend/tgis_backend.py (hunk 1 above)
Comment (gkumbhat) [reply caikit_tgis_backend/tgis_backend.py]: I think what @evaline-ju was referring to was to include `model_id` in the same log message so its easier to track. So something like:
`connection config for model %s :  %s`

Comment (evaline-ju) [reply caikit_tgis_backend/tgis_backend.py]: yes that was what I had intended

Comment (swith004) [reply caikit_tgis_backend/tgis_backend.py]: misunderstood - fixing this @evaline-ju @gkumbhat

**Review by gkumbhat — APPROVED** [review]
Looks good to me!

**Pull request merged**